WordPress/Classes/ViewRelated/Gutenberg/GutenbergImageLoader.swift

```diff
@@ -13,11 +13,11 @@ class GutenbergImageLoader: NSObject, RCTImageURLLoader {
         self.post = post
     }
 
-    func canLoadImageURL(_ requestURL: URL!) -> Bool {
-        return !requestURL.isFileURL
+    func canLoadImageURL(_ requestURL: URL) -> Bool {
+        return requestURL.isFileURL
     }
 
-    func loadImage(for imageURL: URL!, size: CGSize, scale: CGFloat, resizeMode: RCTResizeMode, progressHandler: RCTImageLoaderProgressBlock!, partialLoadHandler: RCTImageLoaderPartialLoadBlock!, completionHandler: RCTImageLoaderCompletionBlock!) -> RCTImageLoaderCancellationBlock! {
+    func loadImage(for imageURL: URL, size: CGSize, scale: CGFloat, resizeMode: RCTResizeMode, progressHandler: RCTImageLoaderProgressBlock, partialLoadHandler: RCTImageLoaderPartialLoadBlock, completionHandler: @escaping RCTImageLoaderCompletionBlock) -> RCTImageLoaderCancellationBlock? {
```

Comment:
Should the return type be optional? AFAICT all of the paths return a block?

Reply:
This method is implemented in `GutenbergImageLoader` to comply with protocol `RCTImageURLLoader`, which defines the method like this (with a `nullable` return type):

```objective-c
- (nullable RCTImageLoaderCancellationBlock)loadImageForURL:(NSURL *)imageURL
                                              size:(CGSize)size
                                             scale:(CGFloat)scale
                                        resizeMode:(RCTResizeMode)resizeMode
                                   progressHandler:(RCTImageLoaderProgressBlock)progressHandler
                                partialLoadHandler:(RCTImageLoaderPartialLoadBlock)partialLoadHandler
                                 completionHandler:(RCTImageLoaderCompletionBlock)completionHandler;
```